cli/js/dispatch.ts

```diff
@@ -74,6 +74,9 @@ export let OP_HOSTNAME: number;
 export let OP_OPEN_PLUGIN: number;
 export let OP_COMPILE: number;
 export let OP_TRANSPILE: number;
+export let OP_BIND_SIGNAL: number;
+export let OP_UNBIND_SIGNAL: number;
+export let OP_POLL_SIGNAL: number;
```

Comment:
Would be better to call these 

OP_SIGNAL_BIND
OP_SIGNAL_UNBIND
OP_POLL_SIGNAL